core/sovereignChainConstants.go

```diff
@@ -0,0 +1,7 @@
+package core
+
+// SovereignChainShardId will be used to identify a shard ID as sovereign chain
+const SovereignChainShardId = uint32(0xFFFFFFFE)
```

Comment:
I think It would be better to have a file containing all shard ID definitions. Something like shardIDConstants.go
Reason: another developer sees the MetachainShardId AllShardId and decides to have create a new constant there that might have the same value as SovereignChainShardId or MainChainShardId

Reply:
But could be modified as you have proposed as it does not have any downside.